samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/autoscrollfallback_tests/ScrollViewWithPaddingPassingTest.kt

```diff
@@ -0,0 +1,71 @@
+package com.kaspersky.kaspressample.autoscrollfallback_tests
+
+import android.Manifest
+import androidx.test.rule.ActivityTestRule
+import androidx.test.rule.GrantPermissionRule
+import com.kaspersky.kaspressample.MainActivity
+import com.kaspersky.kaspressample.screen.MainScreen
+import com.kaspersky.kaspressample.screen.ScrollViewWithPaddingScreen
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import org.junit.Rule
+import org.junit.Test
+
+class ScrollViewWithPaddingPassingTest : TestCase() {
```

Comment:
We could refactor the body of the tests I added, since it is exactly the same as in ScrollViewWithPaddingFailingTest.

They differ only  in the kaspressoBuilder and @Test(expected = error)

Reply:
Could not find a nice way to deal with it.... any idea welcome